index.d.ts

```diff
@@ -27,7 +37,7 @@ interface VerifyRequestByKeyIdInterface {
     signature: string,
     keyId: string,
     requestOptions?: RequestOptions,
-  ): Promise<boolean>;
+  ): Promise<{ isValid: boolean; cacheId: string }>;
```

Comment:
I think we will have to return `{ isValid, cache }`, where `cache` is the object with both the id and the keys. Otherwise I can't get the information I need for subsequent `verifyRequestByKeyId()` calls